### ISSUE TITLE
No feedback when exiting telepresence session

### ISSUE THREAD

**Issue by containscafeine** [telepresenceio/telepresence#594]
I'm running telepresence like -
```console
$ telepresence --verbose --swap-deployment qotm --expose 80:5000 --docker-run -it nginx
Volumes are rooted at $TELEPRESENCE_ROOT. See https://telepresence.io/howto/volumes.html for details.

127.0.0.1 - - [13/Apr/2018:09:28:55 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.59.0" "-"
127.0.0.1 - - [13/Apr/2018:09:28:57 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.59.0" "-"
127.0.0.1 - - [13/Apr/2018:09:28:59 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.59.0" "-"
^C
```

When I exit the session using Ctrl+C, then it takes a couple of seconds for telepresence to terminate, but there is no feedback on the screen that says the process is terminating. Since I'm oblivious to this, I press Ctrl+C again, which immediately stops the execution with a traceback, but it does not restore the state of the Deployment to the original state in the cluster, and I end up with a broken state of my original application.

This is what I'd expect when I send a SIGINT -
```console
$ telepresence --verbose --swap-deployment qotm --expose 80:5000 --docker-run -it nginx
Volumes are rooted at $TELEPRESENCE_ROOT. See https://telepresence.io/howto/volumes.html for details.

127.0.0.1 - - [13/Apr/2018:09:28:55 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.59.0" "-"
127.0.0.1 - - [13/Apr/2018:09:28:57 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.59.0" "-"
127.0.0.1 - - [13/Apr/2018:09:28:59 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.59.0" "-"
^C
Terminating the telepresence session ... done
Shutting down local docker container nginx ... done
Restoring original state of the application ... done
```

**Comment by ark3** [comment]
Thanks for the issue and proposed output. Once #572 is addressed, we should be able to address this and a number of similar issues quickly.

**Comment by ark3** [comment]
This is at least partially addressed in #702, as Tel will now show `Keyboard interrupt (Ctrl-C/Ctrl-Break) pressed`. All the other information in your trace is available and is output to the log file. We should think about whether we want Telepresence to be able to increase verbosity to the console under some circumstances. For example, it might make sense to show this stuff to the user in the SIGINT case.

**Comment by ark3** [comment]
With #886 addressed, I believe this is now fixed.
